packages/manager/modules/iam/src/components/onboarding/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,10 @@
+{
+  "iam_onboarding_title": "Gestion des identités et des accès (IAM)",
+  "iam_onboarding_description": "Gérez les identités de vos utilisateurs et de vos applications, ainsi que leurs droits d’accès dans une interface unique pour tous vos services.",
+  "iam_onboarding_description_2": "OVHcloud IAM permet de gérer finement l’ensemble des autorisations des produits OVHcloud et d’obtenir un meilleur niveau de sécurité avec une vision centralisée de vos accès.",
+  "iam_onboarding_cta_1": "Créer une politique",
+  "iam_onboarding_cta_2": "Créer des utilisateurs",
+  "iam_onboarding_guide_iam_description": "",
```

Comment:
Expected empty value ?

Reply:
It will be added later. We don't have the content yet.